documentation/docs/25-build-and-deploy/95-adapter-aws-sst.md

```diff
@@ -0,0 +1,60 @@
+---
+title: AWS via SST
```

Comment:
I don't think we'd want to add any docs here. We've only added docs for adapters we maintain. We could potentially link to external docs though

Reply:
For comparison, the [adapter-auto docs](https://kit.svelte.dev/docs/adapter-auto) link out to the `svelte-adapter-azure-swa` docs rather than including them on kit.svelte.dev — I think that's the right approach

Reply:
Ah makes sense! Removed.